tools/pyclient/src/molgenis_emx2_pyclient/client.py

```diff
@@ -26,7 +27,7 @@ def __init__(self, url: str, schema: str = None) -> None:
         A Client class instances is created with a server url.
         """
         self._as_context_manager = False
```

Comment:
I vaguely remember a pull request that addressed this issue. It might be worth checking to see if a fix was implemented and potentially remove the context manager(?)